docs/UserGuide.md

```diff
@@ -1,12 +1,16 @@
----
-layout: page
-title: User Guide
----
+<frontmatter>
+  layout: default.md
+  title: "User Guide"
+  pageNav: 3
+  pageNavTitle: "Table Of Content"
+</frontmatter>
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+# User Guide
```

Comment:
```suggestion
# AB-3 User Guide
```
Otherwise the product name is not mentioned in the PDF version.

Jekyll:
![image](https://user-images.githubusercontent.com/1673303/218266661-625b309f-1217-42b9-92cf-7d2fb877d746.png)

MarkBind:
![image](https://user-images.githubusercontent.com/1673303/218266742-124bd523-e5ba-455d-a0b3-ba70bb8b150b.png)